core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/public/modeler.xul

```diff
@@ -495,9 +495,9 @@
                 </columns>
                 <rows>
                   <row flex="1">
-                    <image id="warning-img-6" src="images/spacer.gif" pen:classname="pentaho-warningbutton"/>
+                    <image id="warning-img-6" src="images/spacer.gif" pen:classname="pentaho-warningbutton" pen:imagealttext="${modeler.warning}"/>
                     <label id="memberprops_message_label" flex="1" multiline="true"
-                           value="This is a test to see how a warning message will appear."/>
+                           value="${test_message}"/>
```

Comment:
updated to`${test_message}` to be consistent with similiar labels/images. I do not know the reason behind the wording nor the variable name.